### ISSUE TITLE
Node.js support

### ISSUE THREAD

**Issue by tapz** [simonbengtsson/jsPDF-AutoTable#572]
jsPDF now has a separate distribution file for node.js, but jspdf-autotable does not support that. It adds the add-on to the regular version causing "doc.autoTable is not a function" error . jspdf-autotable needs to load the correct distribution file and add the add-on there when used under node.js.

The node examples work with the regular version but not with the node version. For some reason, however, my own tests cause "ReferenceError: Blob is not defined" with the regular version. Not sure what in my code causes jsPDF to use Blob, but Blob is only available in browsers, not in node.js.

**Comment by rmdias** [comment]
working on it with @ssmiech

**Comment by AminBangash** [comment]
jspdf-auotable is not working in angular 8. Is it not supported in version 8. when i included it in project, it gives no error but no detection of auto-table, just blank page is printed. Can somebody have idea how to use it in angular 8.

**Comment by mmghv** [comment]
Implemented in #612

**Comment by rinkeshjain** [comment]
following code in working on node
```
const { jsPDF } =require( 'jspdf');
require('jspdf-autotable');

const doc=new jsPDF();
doc.autoTable({
    head: [
    ['ID', 'Name', 'Email', 'Country', 'IP-address'],
    ],
    body: [
    ['1', 'HelloäöüßÄÖÜ', 'dmoore0@furl.net', 'China', '211.56.242.221'],
    ['2', 'Janice', 'jhenry1@theatlantic.com', 'Ukraine', '38.36.7.199'],
    ['3', 'Ruth', 'rwells2@constantcontact.com', 'Trinidad and Tobago', '19.162.133.184'],
    ['4', 'Jason', 'jray3@psu.edu', 'Brazil', '10.68.11.42'],
     ['5', 'Jane', 'jstephens4@go.com', 'United States', '47.32.129.71'],
    ['6', 'Adam', 'anichols5@com.com', 'Canada', '18.186.38.37']
    ],
    });
doc.save("abc.pdf");
```
